docs/content/how-to/writing-tables.md

```diff
@@ -262,6 +262,11 @@ filter_spec is equal to the 'WHERE' clause in SQL DELETE statement. Examples:
     id > (SELECT count(*) FROM employee)
 ```
 
+{{< hint info >}}
+If you have set `merge-engine` = `partial-update` or `aggregation`, it will be reset by dynamic options 
```

Comment:
remove this? I cannot understand it from user side.

Reply:
Agree. I think it's not important for users to know.